docs/HowTo/kafka-cluster-migration/sync-topics-from-cluster.md

```diff
@@ -20,15 +20,15 @@ topics from an existing Apache Kafka cluster and assigning them to a Klaw team.
      will now appear in the **Topics** menu, assigned to the chosen
      team.
    - `Bulk Options` tab: Select all topics from the
-     Kafka cluster at once, choose a team and save. The selected
+     Apache Kafka cluster at once, choose a team and save. The selected
      topics will now appear in the **Topics** menu, assigned to the
      chosen team.
 
      ![image](../../../static/images/sync/SyncTopicsFromCluster.png)
 
 3. You can view all the topics and assign topics to a different team by
    selecting the **Show All** option. This displays all the Apache
-   Kafka topics, both assigned and unassigned to any team.
+   Apache Kafka topics, both assigned and unassigned to any team.
```

Comment:
"Apache" is double here, I think that's because there was a line break between "Apache" and "Kafka topics" before. Can you remove the double Apache as well as the line break?